speech/check_dataVals.m

```diff
@@ -157,32 +180,73 @@ function reload_dataVals(src,evt)
     shortTrials = [];
     longTrials = [];
     nanFTrials = [];
-    jumpF1Trials = [];
-    jumpF2Trials = [];
-    fishyF1Trials = [];
+    for s = 1:UserData.nSigs
+        jumpTrials.(UserData.sigs2plot{s}) = [];
+    end
+    fishyTrials = [];
```

Comment:
Separate jumpTrials for each sig2plot, but one fishyTrials -- is it based on the first sig2plot?

Reply:
Correct, fishyTrials currently only evaluates against the first sig2plot. Documented in comments on L43 and L249. Different default values for fishyTrials are set in L49-55 if 'f0' or 'int' is the first signal in sig2plot.